src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.h

```diff
@@ -269,7 +269,10 @@ fail.
 -   Subset of all material specs is currently imported as @ref PhongMaterialData,
     including normal textures
 -   Ambient color is always @cpp 0x000000_rgbf @ce (never a texture)
--   Only the first set of texture coordinates is supported
+-   Unless explicitly enabled by setting the configuration option
+    "allowMaterialTextureCoordinateSets"
+    to @cpp true @ce, only the first set of texture coordinates is supported. See details
+    @ref Trade-TinyGltfImporter-configuration "below".
```

Comment:
```suggestion
-   Unless explicitly enabled with the @cb{.ini} allowMaterialTextureCoordinateSets @ce 
    @ref Trade-TinyGltfImporter-configuration "configuration option", only the 
    first set of texture coordinates is supported.
```

Basically same as in the case of Assimp.